xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/pvr/Timers.h

```diff
@@ -822,7 +866,7 @@ class PVRTimerType : public DynamicCStructHdl<PVRTimerType, PVR_TIMER_TYPE>
   /// @brief **optional**\n
   /// Array containing the possible values of @ref PVRTimer::SetMaxRecordings().
   ///
-  /// @param[in] maxRecordings List of lifetimes values
+  /// @param[in] maxRecordings List of max recordings values
```

Comment:
I assume this was just an error in the docs?

Reply:
Yep, c/p error I guess